packages/node_modules/@node-red/editor-client/locales/en-US/editor.json

```diff
@@ -288,6 +288,17 @@
         "copyMessageValue": "Value copied",
         "copyMessageValue_truncated": "Truncated value copied"
     },
+    "stopstart":{
+        "status": {
+            "state_changed": "Flows runtime has been changed to '__state__' state"
```

Comment:
When the user clicks the 'restart flows' button, they get a notification saying:

 - `Successfully restarted flows`

For the start/stop messages I think it would be more accessible to say:

 - 'Flows started'
 - 'Flows stopped'

rather than talk about states.

Reply:
Will pick this up first thing tomorrow

Reply:
Of course - just cos I'm reviewing at 10pm doesn't mean I expect you to be responding at 10pm 😉

Reply:
Not at all haha.